planty/posts/templates/posts/post_form.html

```diff
@@ -0,0 +1,26 @@
+{% extends "planty/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+<div class="div-content-center form-bg-color">
+    <div class="update-profile-grid">
+        <h1 class="grid-form-title">New Post</h1>
```

Comment:
If this template is shared between adding a new post and editing an existing one, perhaps change the title to `Add / Edit Post` or something like that.